THIS PATCH IS APPLIED TO `calculator/app.py`
 from flask import Flask, request
 import requests
 app = Flask(__name__)
-
+import builtins
 
 @app.route('/checkers', methods=["POST"])
 def checkers():
@@ -33,7 +33,7 @@ def arithmetic():
     if not res:
         return str(result.get('error'))
     try:
-        res_type = type(eval(res))
+        res_type = type(eval(res,  builtins.__dict__, {}))
         if res_type is int or res_type is float:
             return str(res)
         else:
